vendor/src/github.com/coreos/go-systemd/activation/listeners.go

```diff
@@ -24,12 +25,16 @@ import (
 // The order of the file descriptors is preserved in the returned slice.
 // Nil values are used to fill any gaps. For example if systemd were to return file descriptors
 // corresponding with "udp, tcp, tcp", then the slice would contain {nil, net.Listener, net.Listener}
-func Listeners(unsetEnv bool) ([]net.Listener, error) {
+func Listeners(unsetEnv bool, tlsConfig *tls.Config) ([]net.Listener, error) {
```

Comment:
this is changing the vendored files this is not how we update vendors this should be pushed upstream